Build.cmd

```diff
@@ -108,9 +108,9 @@ REM
 REM     NUGET Restore
 REM
 if "%PROJECTPATH%" NEQ "" (
-    call .\Tools\NugetWrapper.cmd restore -NonInteractive %PROJECTPATH%
+	call .\Tools\NugetWrapper.cmd restore -MSBuildPath "%VSINSTALLDIR%\MSBuild\15.0\Bin" -NonInteractive %PROJECTPATH%
```

Comment:
We're using VSWhere inside DevCmd, can we leverage that to get MSBuildPath? This link shows how: https://github.com/Microsoft/vswhere/wiki/Find-MSBuild

And you pointed out that we're calculating a similar path below, can you use vswhere to calculate `MSBUILDPATH`  and update the rest of the script to use a new variable like `MSBUILDEXEPATH`?

Reply:
I could, but I'm not sure it would work in a VS2019 only scenario. A few lines below that, the script is passing VisualStudioVersion="15.0". Vswhere does not provide a VSVersion or something like that. The closest thing I found is "installationVersion", which return 15.9.28307.586 for my 15.9.11.

Reply:
Any ideas?

Reply:
I think it's ok to continue to have that value be 15.0 even when we're using VS2019. I think it was an earlier attempt to fix an issue in Azure Pipelines that turned out to not be needed.